src/components/Text/Base.js

```diff
@@ -33,13 +36,24 @@ const TextBase = ({
   return (
     <Text
       className={classes}
-      size={size}
+      size={{
+        small: responsiveSize.small || size || "hecto",
+        medium:
+          responsiveSize.medium || responsiveSize.small || size || "hecto",
+        large:
+          responsiveSize.large ||
+          responsiveSize.medium ||
+          responsiveSize.small ||
+          size ||
+          "hecto"
```

Comment:
You don't need to fallback on `"hecto"` since it is the default value for the `size` prop